test/testcafe/spec/EnrollAuthenticatorEmail_spec.js

```diff
@@ -158,9 +164,9 @@ test
 
     await t.expect(enrollEmailPageObject.form.getTitle()).eql('Verify with your email');
 
-    const emailAddress = xhrEnrollEmailWithEmailMagicLink.user.value.identifier;
+    // const emailAddress = xhrEnrollEmailWithEmailMagicLink.user.value.identifier;
     await t.expect(enrollEmailPageObject.getFormSubtitle())
-      .eql(`We sent an email to ${emailAddress}. Click the verification link in your email to continue or enter the code below.`);
```

Comment:
v3 widget doesn't display the email address, at least in the tests

Reply:
Do we need to have different assertions for v2 vs v3?

Reply:
I can add that but I was trying to avoid having different assertions for v2 and v3. Maybe a parking lot discussion item?

Reply:
I think we could update the `enrollEmailPageObject` to have a new method that asserts different texts for v2 vs v3 like `hasEmailVerificationSubtitle(emailAddress)` so the branching isn't in the spec itself but rather still in the page object.